resources/lang/en/admin.php

```diff
@@ -7,5 +7,19 @@
     'accept' => 'Accept',
     'reject' => 'Reject',
     'show' => 'Show',
+    'room' => 'Room',
+    'bed1' => 'Bed 1',
+    'bed2' => 'Bed 2',
+    'bed3' => 'Bed 3',
+    'assign_student' => 'Assign a student',
+    'select_floor' => 'Select floor',
+    'select_room' => 'Select room',
+    'first_name' => 'Select name',
+    'second_name' => 'Second name',
+    'starting_date' => 'Accommodation starting date',
+    'finish_date' => 'Accommodation finish date',
```

Comment:
Focus on translating the *thing*, and only the *thing*.

If necessary, hoist the `Accommodation` prefix to a different record.